src/components/Button/Button.tsx

```diff
@@ -1,28 +1,40 @@
 import './Button.css';
 
+import { FC } from 'react';
 import classNames from 'classnames';
 
-const Button = ({
+interface ButtonProps {
+  children: string;
+  disabled?: boolean;
+  light?: boolean;
+  stretch?: boolean;
+  color?: 'primary' | 'secondary' | 'danger';
+  size?: 'md' | 'xl' | 'xxl';
+  type?: 'submit';
```

Comment:
so you're saying that `type` can be only `submit`.
Is it what you wanted?

Reply:
Good point, thanks!